ComponentKit/Core/ComponentTree/CKComponentGroupTreeNode.h

```diff
@@ -0,0 +1,19 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import "CKComponentTreeNode.h"
+
+#include <vector>
+
+@interface CKComponentGroupTreeNode : CKComponentTreeNode
```

Comment:
You need a lengthy docblock for this. Until I get a full explanation, I won't understand fully how all this fits together.